apps/codeforafrica/src/components/GuidingPrinciplesCardList/GuidingPrinciplesCardList.js

```diff
@@ -6,9 +6,9 @@ import GuidingPrinciplesCard from "../GuidingPrinciplesCard";
 
 const GuidingPrinciplesCardList = React.forwardRef(
   function GuidingPrinciplesCardList(props, ref) {
-    const { principles, title, ...other } = props;
+    const { list, title, ...other } = props;
```

Comment:
@kilemensi  isn't `principles` more descriptive? Also, the component already ends with `List`

Reply:
It is but we're not consistent e.g. `OurImpact`, `OurPartners`, etc., seem to be using `title` and `list` while `Articles` seem to be using `articles`, etc. I've just adopted the `title` and `list` here since they'll be suitable in all situations and one doesn't need to remember the prop name.

... kinda like how we switched every html body to just content rather than keep mixing `content` and `description`.

Reply:
I see I see. Makes sense.